packages/server/routes/api/guilds.js

```diff
@@ -4,51 +4,166 @@ const router = express.Router();
 const GuildController = require("../../controllers/guilds");
 const AuthController = require("../../controllers/auth");
 
+// Get all guilds from database
 router.get("/", AuthController.authenticate, async (request, response) => {
   try {
-    const guilds = await GuildController.getAllGuilds();
     response.status(200).json({
       status: "success",
       data: {
-        guilds,
-      },
+        guilds: await GuildController.getAllGuilds(),
+      }
     });
   } catch (error) {
-    response.status(500).json({
+    // Should never error.
+    // In the case that there are no guilds, 
+    // the return would be an empty array.
+    // Here for redudancy.
+    response.status(400).json({
       status: "error",
-      message: error.message,
+      errorName: error.name,
+      errorMessage: error.message,
     });
```

Comment:
Make sure to follow the [JSend](https://app.clickup.com/9011073139/v/dc/8chm43k-2311/8chm43k-2871) format for your API responses. I won't comment on the other ones in your code but make sure they follow the format for the sake of consistency across our entire API.

For this response, it should be changed to
```js
response.status(500).json({
  status: "error",
  message: error.message
});
```

Also for HTTP status code numbers, the 400 range is usually reserved for a "bad request" when the client making the API request makes an invalid request like invalid data. The 500 range is usually reserved for errors that occur server-side.